--- minicom-2.2.orig/src/config.c	2006-04-02 11:52:30.000000000 +0200
+++ minicom-2.2/src/config.c	2006-10-19 17:50:24.000000000 +0200
@@ -54,13 +54,10 @@
       sleep(2);
       return;
     }
-    fprintf(stderr,
-            _("minicom: there is no global configuration file %s\n"), parfile);
-    fputs(_("Ask your sysadmin to create one (with minicom -s).\n"), stderr);
-    exit(1);
+  } else {
+    readpars(fp, CONFIG_GLOBAL);
+    fclose(fp);
   }
-  readpars(fp, CONFIG_GLOBAL);
-  fclose(fp);
   /* Read personal parameters */
   if ((fp = sfopen(pparfile, "r")) != NULL) {
     readpars(fp, CONFIG_PERSONAL);
